src/main/java/com/conveyal/gtfs/loader/StringListField.java

```diff
@@ -14,17 +17,20 @@ public StringListField(String name, Requirement requirement) {
     }
 
     @Override
-    public String validateAndConvert(String original) {
-        // FIXME
+    public ValidateFieldResult<String> validateAndConvert(String original) {
+        // FIXME: This function is currently only used during feed loading. Because the StringListField only exists for the
```

Comment:
Remove FIXME or fix the underlying issue, please.

Reply:
Again if this concrete class does not need to implement a method defined on its interface or superclass, it should throw an unsupported operation exception and include a comment about why it doesn't need to support the operation.